shipgen/MuDISGenerator.h

```diff
@@ -41,12 +41,17 @@ class MuDISGenerator : public FairGenerator
   Double_t startZ,endZ;
   TClonesArray* iMuon ;
   TClonesArray* dPart ;
+  TClonesArray* dPartSoft ;
   FairLogger*  fLogger; //!   don't make it persistent, magic ROOT command
   TFile* fInputFile;
   TTree* fTree;
   int fNevents;
   int fn;
   bool fFirst;
+
+  const Double_t c_light = 29.9792458;//speed of light in cm/ns
+  const Double_t muon_mass = 0.10565999895334244;  //muon mass in GeV
```

Comment:
If these are defined here, they are not needed in the implementation file.
If they are not needed in the header, it would be good to delete them here.

Reply:
They have been removed from the .cxx